README.md

```diff
@@ -37,12 +37,12 @@ Example of correct usage:
 ```asm
     ; RGBASM requires whitespace before the macro name
     struct NPC
-    words 1, YPos         ; 2 bytes
-    words 1, XPos         ; 2 bytes
-    bytes 1, YBox         ; 1 byte
-    bytes 1, XBox         ; 1 byte
-    bytes 6, Name         ; 6 bytes
-    longs 4, MovementData ; 8 bytes
+    words 1, YPos         ;  2 bytes
+    words 1, XPos         ;  2 bytes
+    bytes 1, YBox         ;  1 byte
+    bytes 1, XBox         ;  1 byte
+    bytes 6, Name         ;  6 bytes
+    longs 4, MovementData ; 16 bytes
```

Comment:
I also noticed a typo here so I fixed it in this PR

Reply:
That was an unsynced change, so thanks!